docs/features/exporttopdf/README.md

```diff
@@ -0,0 +1,213 @@
+```
+
+After installing the package, copy or link the `exportpdf` folder from the `ckeditor4-plugin-exportpdf` package to the CKEditor 4 `plugins/` directory.
+
+If you prefer not to touch any files, you may use the [`CKEDITOR.plugins.addExternal()`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_plugins.html#method-addExternal) API method instead to load the Export to PDF plugin directly from the `node_modules/` directory:
+
+```js
+CKEDITOR.plugins.addExternal( 'exportpdf', './node_modules/ckeditor4-plugin-exportpdf/' );
+```
+
+### Custom Build with Online Builder
+
+The Export to PDF plugin is also available in the [official CKEditor 4 Add-ons Repository](https://ckeditor.com/cke4/addon/exportpdf). This allows you to automatically include it in any custom build which is created with the [online builder](https://ckeditor.com/cke4/builder).
+
+### Manual Download
+
+If you prefer to download the plugin code manually, you can do this via the Add-ons Repository. Simply download the [Export to PDF plugin](https://ckeditor.com/cke4/addon/exportpdf) and place it in the CKEditor 4 `plugins/` directory.
+
+## Supported Features and Compatibility
+
+Thanks to the plugin, the formatting and content structure you create using the WYSIWYG editor is maintained in the output PDF file. See the tips for {@link features/exporttopdf/README#achieving-the-best-results achieving the best results} to learn how to configure the plugin and editor to best suite your needs.
+
+As for browser compatibility, the Export to PDF plugin works in all browsers {@link guide/dev/browsers/README supported by CKEditor 4} except for Internet Explorer older than version 11. The plugin is compatible with CKEditor 4 versions starting from `4.6.1`.
+
+## Configuration
+
+The crucial aspect of this feature is its configuration. In order to ensure that the generated PDF looks as close as possible to the same content when it is displayed in the WYSIWYG editor, the feature should be carefully configured. The configuration options available both in the plugin and the HTML to PDF converter allow you to create PDF documents that will fulfill the needs of your application and your end-users.
+
+### Achieving the Best Results
+
+Due to the differences between browsers and operating systems it is not always possible to reach a perfect match between the content in the editor and in a generated PDF file. However, thanks to flexible configuration, adjusting a few configuration options can make the difference hardly noticeable.
+
+Here are a few configuration tips that will make it easier to achieve a close to 1:1 rendering between the editor content and the output PDF document:
+
+* **Editor width.**
+  Make the editor width correspond with the chosen paper format for output file. For example, for the `A4` format (which is used by the PDF converter service [by default](https://pdf-converter.cke-cs.com/docs#section/PDF-options/Page-format)), the editor width should be equal to `805px`.
```

Comment:
A4 is the default format of the converter, while Letter is the most popular US format. It would be cool to have the value for Letter here, too.

Reply:
Added info. However... I just discovered that _something_ (don't know if it's macOS, Chrome or CKEditor) detects if you have a mouse plugged and then modifies the styles on page (scrollbar in particular), which makes it impossible to determine the width that will always work (see the gif below - I'm just plugging and unplugging mouse) :disappointed:  I assumed that more people use mouse than doesn't and both values (for `A4` and `Letter` ) are correct if mouse is connected.

![](https://user-images.githubusercontent.com/10883184/84155611-cfeda480-aa68-11ea-8cad-cc114928d0d5.gif)

Reply:
:woman\_facepalming:  I don't even...

Thanks for researching this! Let's wait for feedback...